device-types/Ruckus/r760.yaml

```diff
@@ -0,0 +1,31 @@
+---
+manufacturer: Ruckus
+model: R760
+slug: ruckus-r760
+part_number: 901-R760-XX00
+u_height: 0.0
+is_full_depth: false
+airflow: passive
+comments: '[Ruckus R750 Datasheet](https://www.commscope.com/product-type/enterprise-networking/wireless-access-points/indoor/r760/)'
+weight: 1.76
+weight_unit: kg
+console-ports:
+  - name: USB
+    type: usb-a
+power-ports:
+  - name: DC
+    type: dc-terminal
+    maximum_draw: 38
+interfaces:
+  - name: Port 1
+    type: 1000base-t
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+    label: 1G Eth
+  - name: Port 2
+    type: 2.5gbase-t
+    label: 2.5G Eth PoE
+    poe_mode: pd
+    poe_type: type1-ieee802.3af
+  - name: WiFi
+    type: ieee802.11ax
```

Comment:
Are you sure that this server has WiFi built-in?

Reply:
Just a Ruckus Access Point - As long as I have it in the right category? Sorry if there is confusion.